data/blockchain/blockchain_test.go

```diff
@@ -13,6 +14,7 @@ func TestNewBlockChain_NilBadBlockCacheShouldError(t *testing.T) {
 
 	_, err := blockchain.NewBlockChain(
 		nil,
+		&statusHandler.NilStatusHandler{},
```

Comment:
Should add a test with nil status handler

Reply:
done